tock/tock/templates/hours/reporting_period_list.html

```diff
@@ -9,10 +9,41 @@
 <div class="reporting-periods">
 	<h3>Reporting Periods You Need to Fill Out</h3>
 	<ul>
-	{% for reporting_period in uncompleted_reporting_periods %}
-	    <li><a href="{% url 'reportingperiod:UpdateTimesheet' reporting_period %}">{{ reporting_period.start_date | date:"F j, Y" }} to {{ reporting_period.end_date | date:"F j, Y" }}</a></li>
-	{% endfor %}
+		{% if uncompleted_reporting_periods %}
+			{% for reporting_period in uncompleted_reporting_periods %}
+	    	<li><a href="{% url 'reportingperiod:UpdateTimesheet' reporting_period %}">
+					{{ reporting_period.start_date | date:"F j, Y" }}
+					 to
+					{{ reporting_period.end_date | date:"F j, Y" }}</a></li>
+			{% endfor %}
+		{% else %}
+			<i>You are up to date!</i>
+		{% endif %}
 	</ul>
+		{% if amendable_completed_reporting_periods %}
+		<h3>Prior Reporting Periods That May be Amended by You</h3>
+			<ul>
+			{% for reporting_period in amendable_completed_reporting_periods %}
+				<li><a href="{% url 'reportingperiod:UpdateTimesheet' reporting_period %}">
+					{{ reporting_period.start_date | date:"F j, Y" }}
+					to
+					{{ reporting_period.end_date | date:"F j, Y" }}</a></li>
+			{% endfor %}
+			</ul>
+		{% endif %}
+		{% if completed_reporting_periods %}
+	<h3>Prior Reporting Periods That May Not be Amended by You</h3>
+		<ul>
+		{% for reporting_period in completed_reporting_periods %}
+			<li><a href="{% url 'reports:ReportingPeriodUserDetailView' reporting_period=reporting_period.start_date username=user.username %}">
```

Comment:
TODO: Test that correct URLs for reporting periods are returned where the reporting periods cannot be edited.